app/src/main/java/keystoretest/com/keystoretest/MainActivity.java

```diff
@@ -24,6 +24,7 @@
     private CheckBox encryptionRequiredCheckBox;
     boolean encryptionRequired;
     private Prefs prefs;
+    private SecretKeyWrapper mSecretKeyWrapper;
```

Comment:
could drop `m` prefix to stick with convention. Appreciate that the local var looks like it had a prefix also, which is even more confusing!